installer/PowerToysSetup/Product.wxs

```diff
@@ -255,6 +271,21 @@
                   DllEntry="UninstallEmbeddedMSIXCA"
                    />
 
+    <CustomAction Id="InstallDSCModule"
+                  Return="ignore"
+                  Impersonate="yes"
+                  Execute="deferred"
+                  BinaryKey="PTCustomActions"
+                  DllEntry="InstallDSCModuleCA"
+                   />
+
+    <CustomAction Id="UninstallDSCModule"
```

Comment:
Question, if Execute is deferred for one of them, why isn't it also for the other one?

Reply:
InstallDSCModuleCA needs to know the location of the install folder to be able to copy the DSC files. For this, I use the `getInstallFolder` helper function. This requires a deferred custom action with a separate custom action to set the parameter.

UninstallDSCModuleCA doesn't need this, so I left it as it is (default=Immediate I believe). I suppose it would still work as a deferred custom action, but I would have to retest it :)